### ISSUE TITLE
rtPip.install fails with envActivation

### ISSUE THREAD

**Issue by EricDales** [jfrog/jenkins-artifactory-plugin#415]
**Describe the bug**
When using envActivation, rtPip.install is failing with message: 
```
java.io.IOException: pip install failed: Cannot run program "source ./venv/bin/activate && pip" (in directory "."): error=2, No such file or directory.
```

**To Reproduce**
Create a pipeline with the following stage:
```groovy
sh 'virtualenv venv'
rtPip.install buildInfo: buildInfo, args: "-r requirements.txt", envActivation: "source ./venv/bin/activate"

```

**Expected behavior**
pip install should ocure in virtualenv venv.

**Versions**
- Jenkins Artifactory plugin version: 3.10.3
- Jenkins operating system: centos (managed with k8s)
- Artifactory Version: 6.6.5

**Additional context**
Looks like the plugin is trying to execute "source ./venv/bin/activate && pip install -r requirements.txt" but considers ""source ./venv/bin/activate && pip" as a single invokation. 
Running env activation and "pip install" thru a shell step works fine.

**Comment by saper** [comment]
Looks like it never could have worked, has this been tested at all? Any working example?

**Comment by yahavi** [comment]
@saper @EricDales 
Thanks for reporting this issue!

As @EricDales said precisly, the root cause is running the following command in Java:
```java
["source ./venv/bin/activate && pip", "install", "-i", "user:password@https://myorg.jfrog.io/artifactory"]
```
To fix it, we will wrap the command with `/bin/bash -c the-command-in-one-string` :
```java
["/bin/bash", "-c", "source ./venv/bin/activate && pip install -i user:password@https://myorg.jfrog.io/artifactory"]
```

Here is the PR that may fix this issue: https://github.com/jfrog/build-info/pull/452

Here is a snapshot with the fix:
https://oss.jfrog.org/oss-snapshot-local/org/jenkins-ci/plugins/artifactory/3.10.x-SNAPSHOT/artifactory-3.10.x-20210203.152343-61.hpi

We'd appreciate your feedback on that.

**Comment by yahavi** [comment]
@EricDales @saper 
Jenkins Artifactory plugin 3.10.5 is released.
This version includes the fix introduced in jfrog/build-info#452. Feel free to upgrade.
We'd appreciate your feedback on that.
